src/components/LabelMenus/styles.module.css

```diff
@@ -1,81 +1,75 @@
 .labelMenu {
-    width: 20%;
+  width: 20%;
 }
 
 .labelMenus {
-    height: 13rem;
-    border: solid 0.0625rem #707070;
-    background-color: #f4f4f4;
-    margin: 2.1875rem 0rem;
-    padding: 1.875rem 0rem 0rem 0rem;
+  height: 13rem;
+  border: solid 0.0625rem #707070;
+  background-color: #f4f4f4;
```

Comment:
You can import these from `layout.css` – they're both in there as `borderDarkGray` and `lightGray`, I believe.